src/Microsoft.OData.Edm/Schema/AmbiguousSingletonBinding.cs

```diff
@@ -22,6 +24,19 @@ public IEdmType Type
             get { return new BadEntityType(String.Empty, this.Errors); }
         }
 
+        public IEdmEntityType EntityType

```

Comment:
Add doc string

Reply:
See this comment https://github.com/OData/odata.net/pull/2887#discussion_r1533480789